docs/rules/arazzo/workflow-dependsOn.md

```diff
@@ -0,0 +1,73 @@
+---
+slug: /docs/cli/rules/arazzo/workflow-dependsOn
+---
+
+# workflow-dependsOn
```

Comment:
Why isn't this named `*-unique` like the others? Is it because we also check it exists?

Reply:
Yes, we also verify that workflows exist.